app.js

```diff
@@ -11,56 +11,74 @@ const totalEl = document.getElementById('total');
 const lossesEl = document.getElementById('losses');
 const winsEl = document.getElementById('wins');
 
-
 let correctGuesses = 0;
 let totalGuesses = 0;
+let incorrectGuesses = 0;
 
 shedButton.addEventListener('click', () => {
     // get a random item to call the 'correct spot'
-
+    const correctSpot = getRandomHidingSpot();
     // call the handleGuess function with the correct parameters (the user's guess and the "correct" hiding place) to do DOM work
+    handleGuess('tree', correctSpot);
```

Comment:
here in your `shedButton` eventListener, it should be checking for `'shed' `as the `userGuess`